app/services/ingest_iati_activities.rb

```diff
@@ -210,8 +210,15 @@ def call
   end
 
   private def add_identifiers(legacy_activity:, new_activity:)
```

Comment:
I'd be interested to hear if you have a preference for inline `private` like this or using a private block - our code base is incosistent in it's use - I keep meaning to see if this is something StandardRB will make a call on so we don't have to?

Reply:
I'm of course happy to follow a linting rule, that would make things more consistent. My preference is for inline ones since I find it a simple and clear way to work out the scope of a method. I do not have to trawl around a class (especially a big one) to make sure the method exists in the right place in the vertical hierarchy.

```
def
private def
protected def
```

Using this syntax methods can appear anywhere in a class and in any order without changing their effective scope:

```
private def
protected def
def
```

For big files I found enough occasions where the pain of having to figuring out a file such as the following was too much needless pain:

```
def 
def
def
def
def
private 
  def
  def
  def (is this in a private scope? I have to scroll up)
def (is this within a scope? I have to scroll up. It's actually meant to be public but has been accidentally reordered)
protected
  def
  def